src/test/java/org/kohsuke/github/ArchTests.java

```diff
@@ -113,14 +138,103 @@ public void testBetaApisAreFlaggedAsDeprecated() {
     @Test
     public void testRequireUseOfAssertThat() {
 
-        String reason = "This project uses `assertThat(...)` instead of other assert*() methods.";
+        final String reason = "This project uses `assertThat(...)` instead of other `assert*()` methods.";
 
-        ArchRule onlyAssertThatRule = methods().that()
-                .haveNameContaining("assert")
-                .should()
-                .haveName("assertThat")
+        final DescribedPredicate<HasName> assertMethodOtherThanAssertThat = nameContaining("assert")
+                .and(DescribedPredicate.not(name("assertThat")));
+
+        final ArchRule onlyAssertThatRule = classes()
+                .should(not(callMethodWhere(target(assertMethodOtherThanAssertThat))))
                 .because(reason);
 
-        onlyAssertThatRule.check(tesetClassFiles);
+        onlyAssertThatRule.check(testClassFiles);
+    }
+
+    @Test
+    public void testRequireUseOfOnlySpecificApacheCommons() {
+
+        final ArchRule onlyApprovedApacheCommonsLang3Methods = classes()
```

Comment:
```suggestion
        final ArchRule onlyApprovedApacheCommonsMethods = classes()
```
`IOUtils` is in commons-io.